lib/bandit/http1/adapter.ex

```diff
@@ -109,6 +116,21 @@ defmodule Bandit.HTTP1.Adapter do
   defp get_version({1, 0}), do: {:ok, :"HTTP/1.0"}
   defp get_version(other), do: {:error, "invalid HTTP version: #{inspect(other)}"}
 
+  defp get_content_length(headers) do
+    with {_, value} <- List.keyfind(headers, "content-length", 0),
+         [length] <- Enum.uniq(PlugUtils.list(value)),
+         length <- String.to_integer(length) do
```

Comment:
This could be `=`

```suggestion
         length = String.to_integer(length) do
```

Reply:
We should be using `Integer.parse/1` here to handle cases without raising. See https://github.com/mtrudel/bandit/pull/56/files#diff-d101e20286894b6a1debc54099265d0118e6edd96abca2dbcd2b7febe7db638fR93-R96

(I wonder if we should be factoring that out into a helper, along with a few of our String.split calls &c?)

Reply:
Switched it to `Integer.parse/2` only accepting value without a remainder.